components/app.js

```diff
@@ -0,0 +1,28 @@
+class App {
+  handleGetGradesError(error) {
+    console.error(error);
+  }
+  handleGetGradesSuccess(grades) {
+    this.gradeTable.updateGrades(grades);
+  }
+  constructor() {
+    this.handleGetGradesError = this.handleGetGradesError.bind(this);
+    this.handleGetGradesSuccess = this.handleGetGradesSuccess.bind(this);
+    this.gradeTable = gradeTable;
+  }
```

Comment:
The general convention is to put the `constructor` before any other method, since that probably needs to be read first.